app/common/Content/Content.scss

```diff
@@ -2,3 +2,10 @@
   flex: 1;
   padding: 0 30px;
 }
+
+.centered {
```

Comment:
Plan on a css refactor later once the groundwork is in. Will need to add media queries for larger screens as well